synfig-studio/src/gui/mainwindow.cpp

```diff
@@ -713,6 +717,21 @@ MainWindow::edit_custom_workspace_list()
 	delete dlg;
 }
 
+void
+MainWindow::refresh_menu_icon_offset()
+{
+	static auto provider = Gtk::CssProvider::create();
+	auto screen   = Gdk::Screen::get_default();
+	const bool is_ltr = App::main_window->get_direction() == Gtk::TEXT_DIR_LTR;
+
+	int icon_offset = -24; // How to discover it in runtime?
```

Comment:
@ice0 please change this to 0 to test on KDE and see if icons are shown

Reply:
Unfortunately no :(

![image](https://user-images.githubusercontent.com/5604544/215277353-e4ef817a-7cb1-44a2-9de0-25be57210ec1.png)